httpclient/httpclient.go

```diff
@@ -6,23 +6,118 @@ Released under MIT license.
 
 package httpclient
 
-import "net/http"
-
-// CloneHTTPRequest creates a shallow copy of the request along with a deep copy of the Headers.
-func CloneHTTPRequest(req *http.Request) *http.Request {
-	r := new(http.Request)
-	*r = *req
-	r.Header = CloneHTTPHeader(req.Header)
-	return r
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/acronis/go-appkit/log"
+)
+
+// New wraps delegate transports with logging, rate limiting, retryable, request id
+// and returns an error if any occurs.
+func New(cfg *Config) (*http.Client, error) {
+	return NewWithOpts(cfg, Opts{})
+}
+
+// Must wraps delegate transports with logging, rate limiting, retryable, request id
+// and panics if any error occurs.
+func Must(cfg *Config) *http.Client {
```

Comment:
Please rename the function constructor from `Must` to `MustNew` for clarity and consistency.